src/test/java/io/orkes/conductor/client/api/MetadataClientTests.java

```diff
@@ -33,6 +34,7 @@ public class MetadataClientTests extends ClientTest {
 
     public MetadataClientTests() {
         metadataClient = super.orkesClients.getMetadataClient();
+        ((OrkesMetadataClient) metadataClient).withReadTimeout(45000);
```

Comment:
Maybe, we should consider renaming `withReadTimeout` to `setReadTimeout`, else this code is looking like Ruby code where the last statement is implicitly the return value.

Reply:
These are following fluent interfaces in java that typically uses `withXXX` and returns the instance as opposed to `setXXX` that returns void.